Categories/UIKit/UIViewController+Promise.swift

```diff
@@ -39,7 +39,7 @@ extension UIViewController {
         return promiseViewController(vc, animate: [.Appear, .Disappear], completion: completion)
     }
 
-    public func promiseViewController<T>(vc: UIViewController, animate animationOptions: PMKAnimationOptions = [.Appear, .Disappear], fulfills: FulfillmentType = .OnceDisappeared, completion: (() -> Void)? = nil) -> Promise<T> {
+    public func promiseViewController<T>(vc: UIViewController, animate animationOptions: PMKAnimationOptions = [.Appear, .Disappear], fulfillmentType: FulfillmentType = .OnceDisappeared, completion: (() -> Void)? = nil) -> Promise<T> {
```

Comment:
Good point. I've added the label back in. The parameter name is still `fulfillmentType`, so that it reads more nicely when used.